getting_started/libraries.adoc

```diff
@@ -146,15 +146,15 @@ After clicking on _Finish_, your first Component Category is ready to be used :)
 [#gettingstarted-libraries-sym]
 ====== Symbol
 
-Now we need to create a Symbol for the OpAmp. So this time you click on the
-_Symbol_ button in the _New Library Element_ wizard. Then fill in some metadata
+Now we need to create a Symbol for the OpAmp. Click on the
+_Symbol_ button in the _New Library Element_ wizard, fill in some metadata
 and click on _Finish_. Now you can place the required pins (choose reasonable
-and unique names!), add the texts `{{NAME}}` and `{{VALUE}}` and draw the
-outline. The result could look like this:
+and unique names!), add the `{{NAME}}` and `{{VALUE}}` variables and draw the
```

Comment:
The "texts" word refers to text elements (which happen to contain variables). What about just using the term "text elements" instead of "variables" above?

```suggestion
and unique names!), add the `{{NAME}}` and `{{VALUE}}` text elements and draw the
```

Reply:
The intent was to point out that those items are not permanent labels but rather placeholders that denote the positioning of the name and value elements.  "texts' on its own isn't a typical  word used to describe the notion of a field or variable.

Reply:
Yes, I get that, but "first place the required pins and then the variables" is not the right terminology, since there are no variable elements. There are text elements that may contain variables though :slightly_smiling_face:

Reply:
ok

Reply:
You can apply the suggestion directly in the webinterface ("Apply Suggestion" or so) :) Or if you like, we can make the adjustments by ourselves.